frame/staking-rewards/src/test/mod.rs

```diff
@@ -162,9 +169,84 @@ fn stake_in_case_of_not_zero_inflation_should_work() {
 		);
 		assert_eq!(<<Test as crate::Config>::Assets as Inspect<<Test as frame_system::Config>::AccountId>>::balance(asset_id, &staker), amount);
 		assert_eq!(<<Test as crate::Config>::Assets as Inspect<<Test as frame_system::Config>::AccountId>>::balance(asset_id, &StakingRewards::pool_account_id(&pool_id)), amount);
+		assert_last_event::<Test, _>(|e| {
+			matches!(e.event,
+            Event::StakingRewards(crate::Event::Staked { pool_id, owner, amount, position_id, ..})
+            if owner == ALICE && pool_id == 1 && amount == 100_500u32.into() && position_id == 1 )
+		});
 	});
 }
 
+#[test]
+fn test_extend_stake_amount() {
+	new_test_ext().execute_with(|| {
+		System::set_block_number(1);
+		assert_eq!(StakingRewards::pool_count(), 0);
+		assert_eq!(StakingRewards::stake_count(), 0);
+
+		let pool_init_config = get_default_reward_pool();
+		assert_ok!(StakingRewards::create_reward_pool(Origin::root(), pool_init_config));
+        let staker = ALICE;
+        let pool_id =  StakingRewards::pool_count();
+        let amount = 100_500u32.into();
+        let extend_amount = 100_500u32.into();
+        let duration_preset = ONE_HOUR;
+        let total_rewards = 100;
+        let total_shares =  200;
+
+		let asset_id = StakingRewards::pools(StakingRewards::pool_count()).expect("asset_id expected").asset_id;
```

Comment:
Why does the formatting look strange here?

Reply:
Not sure , cargo fmt does not show anything.